galasa-ui/src/components/common/BreadCrumb.tsx

```diff
@@ -23,9 +24,18 @@ function BreadCrumb({
   breadCrumbItems: BreadCrumbProps[];
 }) {
   const translations = useTranslations("Breadcrumb");
+  const current = useTheme().theme;
+  const theme: 'g10' | 'g90' = 
+    current === 'light'
+      ? 'g10'
+      : current === 'dark'
+        ? 'g90'
+        : window.matchMedia('(prefers-color-scheme: dark)').matches
+          ? 'g90'
+          : 'g10';
```

Comment:
Minor - this is a bit tricky to read, might be better to define `theme` as a variable with `let` and then use if-else statements?